test/docfx.Tests/DocsetTest.cs

```diff
@@ -217,4 +244,164 @@ public static async Task Build_With_RedirectUri_Files()
         var urls = XDocument.Parse(sitemapXml).Root.Elements();
         Assert.True(urls.Count() == 0);
     }
+
+    [Fact]
+    public static async Task Pdf_Basic()
+    {
+        var outputs = await Pdf(new()
+        {
+            ["docfx.json"] =
+                """
+                {
+                "pdf": {
+                    "content": [
+                        {
+                            "files": [
+                                "**/**.md"
+                            ]
+                        },
+                        {
+                            "files": "**/toc.yml"
+                        }
+                    ],
+                    "wkhtmltopdf": {
+                        "additionalArguments": "--enable-local-file-access"
+                    },
+                    "dest": "_pdf"
+                }
+                }
+                """,
+            ["pdf/toc.yml"] = "- name: Introduction\r\n  href: intro.md\r\n- name: Another Page\r\n  href: anotherpage.md\r\n",
+            ["pdf/intro.md"] = "# Introduction\r\n\r\n",
+            ["pdf/anotherpage.md"] = "# Another Page\r\n\r\n"
+        });
+
+        Assert.True(outputs.ContainsKey("Pdf_Basic_pdf.pdf"));
+    }
+
+    [Fact]
+    public static async Task Pdf_With_Cover_Page()
+    {
+        var outputs = await Pdf(new()
+        {
+            ["docfx.json"] =
+                """
+                {
+                "pdf": {
+                    "content": [
+                        {
+                            "files": [
+                                "**/**.md"
+                            ]
+                        },
+                        {
+                            "files": "**/toc.yml"
+                        }
+                    ],
+                    "wkhtmltopdf": {
+                        "additionalArguments": "--enable-local-file-access"
+                    },
+                    "dest": "_pdf"
+                }
+                }
+                """,
+            ["pdf/toc.yml"] = "- name: Introduction\r\n  href: intro.md\r\n- name: Another Page\r\n  href: anotherpage.md\r\n",
+            ["pdf/cover.md"] = "# My Basic Cover Page\r\n\r\n",
+            ["pdf/intro.md"] = "# Introduction\r\n\r\n",
+            ["pdf/anotherpage.md"] = "# Another Page\r\n\r\n"
+        });
+
+        Assert.True(outputs.ContainsKey("Pdf_With_Cover_Page_pdf.pdf"));
+    }
+
+    [Fact]
```

Comment:
Would this PDF tests pass on linux and mac? If not, we may want to skip them using conditional facts.

Reply:
are you intending to adopt arcade? there's a reusable fact in there https://github.com/dotnet/sdk/blob/a30e465a2e2ea4e2550f319a2dc088daaafe5649/src/Tests/Microsoft.NET.TestFramework/Attributes/WindowsOnlyFactAttribute.cs

Reply:
To me, it seems simpler to copy the code as there aren't a lot of lines there.